packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -85,22 +89,26 @@ export const RelationInput = ({
                     </button>
                   }
                 >
-                  {href ? (
-                    <BaseLink disabled={disabled} href={href}>
-                      {title}
-                    </BaseLink>
-                  ) : (
-                    title
-                  )}
+                  <Box paddingTop={1} paddingBottom={1}>
```

Comment:
@gu-stav I added this padding because when there is no state badge (in case D&P doesn't exist for an entry) the all card padding change